Add the dataset of JHOVE modules

Dhwaniartefact opened the pull request:
This PR includes a sample dataset for JHOVE modules to facilitate file validation.

Connected to: https://github.com/archivematica/Issues/issues/1723

replaceafill committed:
Add files for validating with JHOVE modules

Dhwaniartefact committed:
Add README.md for JhoveModuleValidation dataset

replaceafill commented on SampleTransfers/JHOVEModulesValidation/README.md:
```diff
@@ -0,0 +1,37 @@
+| **TIFF-HUL**     | pass.tif        | [Original](https://github.com/openpreserve/jhove/raw/refs/heads/integration/test-root/corpora/examples/modules/TIFF-hul/little-endian.tif)                    |
+|                  | fail.tif        | [Original](https://github.com/openpreserve/jhove/raw/refs/heads/integration/test-root/corpora/examples/modules/TIFF-hul/badfiles/peppers.tif)                 |
+| **WARC-KB**      | pass.warc       | [Original](https://gist.github.com/replaceafill/e9c10cff981baddeba1ba3ce7e2f25f3)                                                                             |
+|                  | fail.warc **    | Original                                                                                                                                                      |
```
I think we can remove `Original` here.

replaceafill reviewed (approved):
Thank you @Dhwaniartefact.

replaceafill committed:
Use permalinks to original files

Using permalinks to the main GitHub pages of the original files
provides users with greater flexibility, allowing them to either
preview or download the files as needed.

replaceafill commented on SampleTransfers/JHOVEModulesValidation/README.md:
```diff
@@ -0,0 +1,37 @@
+
+## File Summary
+
+| JHOVE Module     | Validation File                                                                                                                                                               |
```
In my opinion the `Source` column makes clear we're taking the files from somewhere else. What is the advantage of removing it?

Dhwaniartefact replied:
Ok. In place of "Original", How should I rename it?

replaceafill replied:
> Ok. In place of "Original", How should I rename it?

Sorry, I'm not sure I understand the question. Could you please clarify?

Dhwaniartefact replied:
> I think we can remove `Original` here.

As per your suggestions, how should I called that file?

replaceafill replied:
I proposed to remove the text `Original` because it was not a link, just text.

replaceafill replied:
I didn't mean to remove the `Source` column.

replaceafill replied:
See how in my commit it is removed https://github.com/artefactual/archivematica-sampledata/pull/85/commits/d8d7ca87245d658c53776dcfdbfacbed9b898df5

Dhwaniartefact replied:
oh...............sorry I got it now. Let me drop last commit now.

replaceafill commented on SampleTransfers/JHOVEModulesValidation/README.md:
```diff
@@ -0,0 +1,37 @@
+| **WARC-KB**      | [pass.warc](https://gist.github.com/replaceafill/e9c10cff981baddeba1ba3ce7e2f25f3)                                                                                            |
+|                  | fail.warc \*\*                                                                                                                                                                |                                                                                                                                                                                                 |
+| **WAVE-HUL**     | [pass.wav](https://github.com/openpreserve/jhove/blob/7306ffaf773577ba34abb21157c1cea38e219731/test-root/corpora/examples/modules/WAVE-hul/8-Bit-Noise-1.wav)                 |
+|                  | [fail.wav](https://github.com/openpreserve/jhove/blob/7306ffaf773577ba34abb21157c1cea38e219731/test-root/corpora/errors/modules/WAVE-hul/wf-pcm-44khz-8bit-mono-chunk-id-above-valid-ascii.wav) |
```
Could we make all rows in the table the same width like it was in the previous commit?
